dali/operators/CMakeLists.txt

```diff
@@ -80,6 +79,8 @@ if (NOT OPERATOR_TEST_SRCS_PATTERN STREQUAL "" OR NOT OPERATOR_TEST_SRCS_PATTERN
                 OPERATOR_TEST_SRCS_PATTERN_EXCLUDE)
 endif()
 
+# Required for operators.cc to have all symbols
+list(APPEND DALI_OPERATOR_SRCS "${CMAKE_CURRENT_SOURCE_DIR}/../nvjpeg/nvjpeg_helper.cc")
```

Comment:
Is there a way to pritify that?

Reply:
Why moving it away from L61?
I think it was put inside:
```
if (NOT OPERATOR_SRCS_PATTERN STREQUAL "" OR
    NOT OPERATOR_SRCS_PATTERN_EXCLUDE STREQUAL "")
```
for a purpose.

Reply:
I guess initially it was just a matter of convienience: not to have to pass it explicity when building subset of ops as it cannot be omitted. Now, it is needed because the file is no longer in any op subdir so it is not discovered with collect_sources.